### DISCUSSION TITLE
Error with current version of app (dev mode)

### DISCUSSION THREAD

**Discussion by vaggelisk** [requarks/wiki#5759] [Error / Bug Report] [answered]
When you you clone a dev instance by

```
git clone https://github.com/requarks/wiki.git  ./<your_folder>

cd  /<your_folder>
```
then on Dashboard View the version is not the 2.5.288, as the version of Latest Stable Release, but is the 2.0.0, as you can see in the image attached.

I understand that it's a  very minor problem, but 

![image](https://user-images.githubusercontent.com/18349081/196374145-b9dc1ba8-eb9f-4ade-813b-6d965e0c6bf9.png)

### Solution

It needs to change `package.json`:

`"version": "2.0.0"` to become `"version": "2.5.288"`

and to change every time there is a new Release.

**Comment by NGPixel** [comment, marked as answer]
Not a bug, this is the expected version in dev. The final version is only injected during build.
